epub33/rs/index.html

```diff
@@ -309,24 +309,14 @@ <h4>File URLs</h4>
 			<section id="sec-epub-rs-conf-xml">
 				<h4>XML processing</h4>
 
-				<p>A reading system MUST be both of the following:</p>
+				<p id="confreq-rs-xml-nval" data-tests="#pub-xml-non-validating_invalid,#pub-xml-non-validating_unclosed">
+                                A reading system MUST use a non-validating XML processor [xml].</p>
 
-				<ul class="conformance-list">
-					<li>
-						<p id="confreq-rs-xml-nval"
-							data-tests="#pub-xml-non-validating_invalid,#pub-xml-non-validating_unclosed">a <a
-								data-cite="xml#proc-types">conformant non-validating XML processor</a> [[xml]].</p>
-					</li>
-					<li>
-						<p id="confreq-rs-xml-ns" data-tests="#pub-xml-names">a <a
-								data-cite="xml-names#ProcessorConformance">conformant processor</a> as defined in
-							[[xml-names]].</p>
-					</li>
-				</ul>
+				<p id="confreq-rs-xml-extid" data-tests="#pub-xml-external-id">It MUST NOT read <a data-cite="xml#dt-doctype>external 
```

Comment:
Isn't "it" here the xml processor, not the reading system? It might be simpler to keep this as a list:

> A reading system MUST use a non-validating XML processor[[xml]] that:
> - does not read external DTD subsets [[xml]];
> - conforms to [[xml-names]].